### ISSUE TITLE
If a new member has applied for a scholarship, they shouldn't get dues setup reminder emails

### ISSUE THREAD

**Issue by brittag** [doubleunion/arooo#528]
Currently if a new member applies for a dues scholarship and is waiting to hear back about it, the Arooo system doesn't know that they're waiting on something (they look like they just haven't finished setting up their account), so they keep getting confusing email reminders about setting up their account.

I wrote some text changes for the email reminders at https://github.com/doubleunion/arooo/pull/527 to reduce that confusion, but really we shouldn't email people at all if (1) they have a Google-friendly email on file, (2) they haven't set up their dues, (3) they _have_ submitted a request for a scholarship.

This is related to https://github.com/doubleunion/arooo/issues/468 - we would need to track which membership have applied for scholarships.

**Comment by brittag** [comment]
Closing as a duplicate of https://github.com/doubleunion/arooo/issues/346
